README.md

```diff
@@ -96,7 +97,7 @@ N/A.
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **Query and update animals table**
+- [ ] **Query multiple tables**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Great job working on this milestone and updating the ReadMe. Unfortunately, when I click on the MIT license link, it gives me an error 404 page. It is not professional to allow broken links in our codebase. Kindly fix this issue.
![image](https://github.com/CarlosIgreda/Vet-clinic-database/assets/62849872/335ebbfd-c666-4616-a0db-07daca5487dd)

Reply:
@t-yanick Done it. The license file was added.